dali/operators/input/video_input_test.cc

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
+#include <fstream>
```

Comment:
The file wouldn't compile after I've removed `#include <fstream>` from somewhere...